src/test/java/net/snowflake/client/jdbc/ConnectionManual.java

```diff
@@ -134,61 +137,62 @@ private void testSSO() throws Throwable {
     System.out.println(
         "[INFO] 1st connection gets id token and stores in the cache file. "
             + "This popup a browser to SSO login");
-    Connection con1 = DriverManager.getConnection(url, properties);
-    assert database.equalsIgnoreCase(con1.getCatalog());
-    assert schema1.equalsIgnoreCase(con1.getSchema());
-    con1.close();
+    try (Connection con1 = DriverManager.getConnection(url, properties)) {
+      assert database.equalsIgnoreCase(con1.getCatalog());
```

Comment:
let's replace with `assertTrue` (also in other changed tests - use `assertEquals` where possible)

Reply:
Changed.